lib/bolt/applicator.rb

```diff
@@ -125,7 +125,16 @@ def compile(target, scope)
           @logger.send(bolt_level, "#{target.name}: #{message}")
         end
       end
-      raise ApplyError.new(target.name, result['message']) unless stat.success?
+
+      unless stat.success?
```

Comment:
This LGTM! Adding a test for it would be awesome.

Reply:
Updated.